modules/ROOT/pages/use-anypoint-monitoring.adoc

```diff
@@ -0,0 +1,115 @@
+= Using Anypoint Monitoring
+
+Mule apps and API gateways deployed to Runtime Fabric include native support for Anypoint Monitoring to allow viewing metrics within Anypoint Platform.
+
+== Anypoint Monitoring
+
+xref:monitoring::index.adoc[Anypoint Monitoring] includes tools that provide feedback from Mule flows and components in your Mule app network. Anypoint Monitoring runs as a sidecar in application deployments and gathers metrics for Mule apps and API gateways deployed to Runtime Fabric. Titanium-plan customers have access to additional capabilities, including application log retrieval and search.
+
+== Enable Anypoint Monitoring
+
+By default, Runtime Fabric is configured to send application metrics to Anypoint Monitoring. Data is securely transmitted using mutual TLS encryption. For metrics and logs to be sent to Anypoint Monitoring, outbound network connectivity must be available. 
+
+If you need an outbound proxy, configure Anypoint Runtime Fabric to use a SOCKS5 proxy by following the instructions in xref:manage-proxy.adoc[Manage Proxies Used by Runtime Fabric].
+
+To verify outbound network connectivity on Anypoint Runtime Fabric version 1.8.50 or later, ensure that each node in Runtime Fabric can send outbound TCP (Lumberjack) communication using port 443 to the following domains:
+
+* US control plane: `dias-ingestor-router.us-east-1.prod.cloudhub.io` 
+* EU control plane: `dias-ingestor-router.eu-central-1.prod-eu.msap.io`
+
+[NOTE]
+====
+If you are using a version of Anypoint Runtime Fabric older than 1.8.50, use port 5044 instead of port 443. Use the following domains:
+
+* US control plane: `dias-ingestor-nginx.prod.cloudhub.io` 
+* EU control plane: `dias-ingestor-nginx.prod-eu.msap.io`
+====
+
+== Metrics
+
+Each Mule app and API gateway replica runs a metrics agent that collects and transmits metrics data to 
+Anypoint Monitoring via secure mutual TLS encryption. Each replica can cache up to 100 megabytes of metrics, after which the metrics data is rotated to provide storage for more recent data. In the event that metrics cannot be sent to Anypoint Monitoring, this typically provides at least 4 days worth of storage per replica. Note that custom metrics might consume more storage.
+
+[NOTE]
+--
+The following Anypoint Runtime Fabric updates do not impact or restore monitoring metrics of applications deployed on Runtime Fabric:
```

Comment:
```suggestion
The following Anypoint Runtime Fabric updates do not impact the or restore monitoring metrics of applications deployed on Runtime Fabric:
```

Reply:
What do you mean that the updates don't restore the monitoring metrics? If they don't impact the metrics, wouldn't they not restore them as well?

Reply:
Confirmed that metrics are owned by control plane. rtf backup restore is a runtime plane operation.